CHANGELOG.md

```diff
@@ -4,115 +4,160 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+Package streamr-broker follows its own versioning and has its own changelog
+found [here](packages/broker/CHANGELOG.md).
 
 ## [Unreleased]
 
-### Added
+### streamr-client
 
-### Changed
+#### Added
 
-### Deprecated
+#### Changed
 
-### Removed
+#### Deprecated
 
-### Fixed
+#### Removed
 
-### Security
+#### Fixed
+
+#### Security
+
+### @streamr/cli-tools
```

Comment:
I'd use simpler header (not the package name) here and when we whenever we refer to cli-tools:

```suggestion
### cli-tools
```